packages/integration-tests/sfnStep.js

```diff
@@ -129,11 +128,16 @@ class SfnStep {
     const subStepExecutionDetails = scheduleEvent.lambdaFunctionScheduledEventDetails;
     let stepInput = JSON.parse(subStepExecutionDetails.input);
 
+    if (stepInput.cma) {
+      stepInput = { ...stepInput, ...stepInput.cma, ...stepInput.cma.event };
+      delete stepInput.cma;
+      delete stepInput.event;
```

Comment:
So we need to include the contents of `stepFunction.cma` because it includes the replace config that is used below on line 140 when we actually pull the remote step function event, if necessary

Reply:
I'm pretty much asking the same here as above, so resolving